app/templates/components/ui-table/header-row-sorting.hbs

```diff
@@ -0,0 +1,34 @@
+<tr>
+  {{#each processedColumns as |column|}}
+    {{#if column.isVisible}}
+      {{#if column.useSorting}}
+        <th class="{{classes.theadCell}}">
+          <a class="ui" href="#" {{action "sort" column}}>
```

Comment:
Let this not be an anchor. Make it a div or span that has an attribute `role="button"`